packages/dom-membrane/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './browser-realm';
+export * from './window';
```

Comment:
why is this re-exported? who is using anything other than default?

Reply:
Actually it was unnecessary, removed it, not sure why I had it 🤷🏻‍♂️